### DISCUSSION TITLE
Disable "Cancel" button for read-only users on accepted GIN/GDN

### DISCUSSION THREAD

**Discussion by alexgrndprx** [YetiForceCompany/YetiForceCRM#14315] [How to] [answered]
Is there a way to disable this red button at top-right corner? I made full read-only Guest profile on gitstable.yetiforce.com and still it can cancel any GIN/GDN/... etc
I just updated, so don't know is this a bug or I just can't find a way to disable it. Thanks

![cancel1](https://user-images.githubusercontent.com/79980057/109806022-d8a48d00-7c2c-11eb-8773-d75a1a5c2c40.PNG)
![cancel2](https://user-images.githubusercontent.com/79980057/109806015-d7736000-7c2c-11eb-8e3b-30ae7bf458c6.PNG)

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
Check table s_yf_record_quick_changer
